### DISCUSSION TITLE
Companies / Sites using Swiper

### DISCUSSION THREAD

**Discussion by nolimits4web** [nolimits4web/swiper#4428] [Show and tell]
If you're using Swiper at your company/organization you can share the URL here.

**Comment by vltansky** [comment]
Some companies / site using swiper:

- Zoom - https://zoom.us
- Imdb - https://www.imdb.com/
- Slack - https://slack.com/intl/en-gb/
- Netflix - https://makeit.netflix.com
- Microsoft - https://www.microsoftstore.com.cn/
- Nvidia - https://nvidia.com
- Ikea - https://www.ikea.cn/cn/en/
- Sony - https://www.sony.co.jp/
- Lenovo - https://www.lenovo.com.cn/
- Booking - https://business.booking.com
- HBO - https://www.hbo.com/
- freepik - https://freepik.com
- HP - https://www8.hp.com/uk/en/home.html
- Xiaomi - https://www.mi.com/uk/
- Huawei - https://huawei.ru/
- Meizu - https://www.meizu.com/
- Cnet - https://japan.cnet.com
- CBS - https://www.cbs.com/
- Cbsnews - https://mobilecms.cbsnews.com
- euronews - https://euronews.com
- Leagueoflegends - https://euw.leagueoflegends.com/en-gb/
- Semrush - https://www.semrush.com/
- ManyChat - https://manychat.com/
- Wpengine - https://rflktlive.wpengine.com
- Casio - https://casio.com
- Billboard - https://www.billboard.com/
- Harvard - https://www.harvard.edu/
- Fastcompany - https://www.fastcompany.com/
